clang/test/CIR/CodeGen/atomic.cpp

```diff
@@ -337,4 +337,16 @@ void incdec() {
 
 // LLVM-LABEL: @_Z6incdecv
 // LLVM: atomicrmw add ptr {{.*}}, i32 {{.*}} monotonic, align 4
-// LLVM: atomicrmw sub ptr {{.*}}, i32 {{.*}} monotonic, align 4
\ No newline at end of file
+// LLVM: atomicrmw sub ptr {{.*}}, i32 {{.*}} monotonic, align 4
+
+void inc(int* a) {
+  int b = __sync_fetch_and_add(a, 1);
+}
```

Comment:
Can you also please add more tests like the ones for `__sync_fetch_and_add` in `clang/test/CodeGen/Atomics.c`?